src/jbake/content/officers-and-board.html

```diff
@@ -82,24 +138,33 @@ <h5 class="card-title" style="text-align: center">Director of Sponsorship</h5>
+                        <a tabindex="0" class="btn btn-lg btn-primary" role="button" data-toggle="popover" data-html=true data-trigger="focus" data-placement="bottom" data-content="">Josh Juneau</a>
                     </div>
                 </div>
             </div>
 
 
             <div class="card text-white bg-primary mb-3" style="max-width: 15rem;">
                 <div class="card-body">
-                    <h5 class="card-title" style="text-align: center">CJUG Alumni</h5>
-                    <img src="images/cjug_small.jpeg" onload="this.onload=null; this.src=gravatar('freddy.guime@cjug.org', '150');" style="display: block;margin-left: auto;margin-right:auto"/>
+                    <h5 class="card-title" style="text-align:center">Board Member</h5>
+                    <img src="images/cjug_small.jpeg" onload="this.onload=null; this.src=gravatar('todd.ginsberg@cjug.org', '150');" style="display: block;margin-left: auto;margin-right:auto"/> 
                     <br/>
                     <div style="text-align:center">
-                        <a tabindex="0" class="btn btn-lg btn-primary" role="button" data-toggle="popover" data-html=true data-trigger="focus" data-placement="bottom" data-content="Freddy is a Senior Developer at Expedia. Always dealing with performance and usability he is always curious on how to make the overabundance of data useful for travelers, traders and consumers. Having worked with different technologies before has allowed him to come with solutions to rendering bottleneck problems. A Usability Guru, Freddy understands and bridges the concepts of high-throughput with usability within software.
-                                <br/><br/>
-                                He is also the author and maintainer of the javapubhouse.com, a podcast dedicated to tutorial topics in Java that covers everything from the use of the keyword volatile to the definition of beautiful code, also of javaoffheap.com, a java news podcast">Freddy Guime</a>
+                        <a tabindex="0" class="btn btn-lg btn-primary" role="button" data-toggle="popover" data-html=true data-trigger="focus" data-placement="bottom" data-content="Todd Ginsberg is a Principal Software Developer at Netspend, a payments company in Austin, TX where he is responsible for prototypes, experiments, and proofs of concept. Todd has been writing in Java since 1995, and Kotlin since 2016. Todd lives in Chicago with his wife, daughter, and two dogs. When not programming, he enjoys reading, walking/hiking, doughnuts, and Formula 1.">Todd Ginsberg</a>

```

Comment:
Would you mind striking the entire sentence that starts with "Todd lives in Chicago..." please?

Reply:
Thanks Todd for the approval !  Yeah, I have updated it.